src/eonstor_ds_cli/common_cli.py

```diff
@@ -1858,17 +1850,44 @@ def retype(self, ctxt, volume, new_type, diff, host):
 
             return (rc, model_update)
         else:
-            if ('infortrend_provisioning' in diff['extra_specs'] and
-                    (diff['extra_specs']['infortrend_provisioning'][0] !=
-                        diff['extra_specs']['infortrend_provisioning'][1])):
+            if ('infortrend:provisioning' in diff['extra_specs'] and
```

Comment:
Please extra variable for infortrend:provisioning and infortrend:tiering. And this can be a checking function.